config.json

```diff
@@ -1,6 +1,7 @@
 {
   "slug": "rust",
   "language": "Rust",
+  "ignore_pattern": "example|description.md|metadata.yml",
```

Comment:
another fun fact: When you need to put a backslash in json string you need two, so the correct regex still looks odd.
`"ignore_pattern": "example|description\\.md|metadata\\.yml"`